### PR TITLE
Fix for node frames

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: translators/materials.py [hunk 1]
```diff
@@ -176,9 +178,10 @@ def _reset(self, mat):
     def __set_shader_group_parameters(self, scene):
         surface_shader = None
         for shader in self.__shaders:
-            if shader.node_type == 'osl_surface':
-                surface_shader = shader
-                self.__shader_list = surface_shader.traverse_tree()
+            if not isinstance(shader, bpy.types.NodeInternal):
+                if shader.node_type == 'osl_surface':
+                    surface_shader = shader
+                    self.__shader_list = surface_shader.traverse_tree()
```
Comment (dictoon) [inline translators/materials.py]: Shouldn't you break the `for` loop when you found what you were looking for?

File: render/__init__.py [hunk 2]
```diff
@@ -235,6 +235,7 @@ def __start_final_render(self, scene, project):
 
         self.__renderer = asr.MasterRenderer(project,
                                              project.configurations()['final'].get_inherited_parameters(),
+                                             [],
```
Comment (dictoon) [inline render/__init__.py]: How did it work until now?

File: translators/materials.py (hunk 1 above)
Comment (jdent02) [reply translators/materials.py]: Probably....

File: render/__init__.py (hunk 2 above)
Comment (jdent02) [reply render/__init__.py]: Master hasn't been updated in a while.  The last time was before Esteban's OSL source code merge, which is when that additional 'resource path' was added.

Comment (dictoon) [reply render/__init__.py]: Oh ok, got it. Thanks.

File: translators/materials.py (hunk 1 above)
Comment (dictoon) [reply translators/materials.py]: Do you want to fix this before merging the PR?

**Comment by jdent02** [comment]
Yep. Should get to it today

**Pull request merged**